### ISSUE TITLE
Wrong sorting of products in taxon after saving

### ISSUE THREAD

**Issue by damonsson** [Sylius/Sylius#11218]
**Sylius version affected**: 1.6 / Any

**Description**  
Wrong sorting of products in taxon after saving. There is no rule, the sorting after saving is random. Sometimes the first saving does not cause errors, sometimes the next one. 

**Steps to reproduce**  
Before save:
![Before save](http://i.imgur.com/NOGck0f.png)
After save:
![Before save](http://i.imgur.com/syogyp6.png)

**Possible Solution**  
Looks like problem with https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/CoreBundle/Resources/config/doctrine/model/ProductTaxon.orm.xml#L30 . Don't have any possible solution.

**Comment by lchrusciel** [comment]
I can confirm, that I could reproduce it on our master demo. Can you write a behat scenario to cover this case? Also, have you found some solution to this problem in the meantime?

**Comment by svonian** [comment]
Can confirm.

It seems this issue stems from the Sortable in StofDoctrineExtensionsBundle

https://symfony.com/doc/current/bundles/StofDoctrineExtensionsBundle/index.html

There's an issue as well https://github.com/Atlantic18/DoctrineExtensions/issues/1642 if you disable the sortable it "fixes it" to a degree.

In your services config put

```
stof_doctrine_extensions:
    orm:
        default:
            sortable: false
```

**Comment by retokohli** [comment]
Any updates on this?

I just opened v1.12.9 and was able to produce similar behavior.  Positions 1..15 save (mostly) correct, but higher ordinals are almost always set wrongly (e.g., number 16 repeats until the end of the list).

**Comment by jakubtobiasz** [comment]
This issue has been resolved in #15225 and will be released along with Sylius `1.12.11`.
